certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -2832,7 +2724,7 @@
                   "description": "secret",
                   "title": "",
                   "x-ms-visibility": "internal",
-                  "default": "MSAutomateProdPur"
+                  "default": "MSAutomate"
```

Comment:
@RjImpx I am not sure whether you are aware of it or not, there is no word like `MS Automate` in our product. It was renamed as `Power Automate` long back. If this value appears to public, it need to be changed to `PowerAutomate`.

Reply:
@sriyen-msft we will fix this, I have sent you my comments on isvstudio, can we connect?

Reply:
> @sriyen-msft we will fix this, I have sent you my comments on isvstudio, can we connect?

I have replied to your question in the ISV Studio